sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -463,6 +463,7 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * <li>`columnNameOfCorruptRecord` (default is the value specified in
    * `spark.sql.columnNameOfCorruptRecord`): allows renaming the new field having malformed string
    * created by `PERMISSIVE` mode. This overrides `spark.sql.columnNameOfCorruptRecord`.</li>
+   * <li>`wholeFile` (default `false`): parse one record, which may span multiple lines.</li>
```

Comment:
In JSON, when `wholeFile` is on, we only parse one and only one record (i.e., table rows). The semantics are different. Let us fix the option name for CSV in 2.2.